examples/functions-with-defaultable-parameters/tests/functions_with_defaultable_parameters_test.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/test;
+import ballerina/io;
+
+any[] outputs = [];
+int counter = 0;
+
+// This is the mock function which will replace the real function
+@test:Mock {
+    moduleName: "ballerina/io",
+    functionName: "println"
+}
+public function mockPrint(any... s) {
+    outputs[counter] = <string>s[0] + <string>s[1] + <string>s[2] + <string>s[3] + <string>s[4] + <string>s[5];
```

Comment:
Shall we use `string.convert` here? With the upcoming changes to the type cast expression, this would fail, since conversion would not happen from numeric basic type to string with `<string>`.

```suggestion
    outputs[counter] = string.convert(s[0]) + string.convert(s[1]) + string.convert(s[2]) + string.convert(s[3]) + string.convert(s[4]) + string.convert(s[5]);
```